sdk/python/feast/cli.py

```diff
@@ -896,6 +896,28 @@ def init_command(project_directory, minimal: bool, template: str):
     show_default=True,
     help="Number of worker",
 )
+@click.option(
+    "--threads",
+    "-t",
+    type=click.INT,
+    default=1,
+    show_default=True,
+    help="Number of thread"
```

Comment:
could we add a bit verbose help especially for the beginners. when to increase threads or how it impacts? or may be reference to unicorn documentation.

Reply:
@lokeshrangineni 
yes. I [commit it](https://github.com/feast-dev/feast/pull/4693/commits/dbd3b3843b0b0b10d8774e9f688ddde17df984b6) more. A documentation link is so long, it may worsen visibility of source code, so I wrote the summary

Reply:
And short-cut of `thread` options doesn't exist in [official gunicorn documentation](https://docs.gunicorn.org/en/stable/settings.html#threads). So, I remove this it, too